jupyterhub/templates/scheduler/rbac.yaml

```diff
@@ -0,0 +1,66 @@
+{{ if .Values.scheduler.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Chart.Name }}-{{ .Release.Name }}-scheduler
+{{ if .Values.rbac.enabled }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Chart.Name }}-{{ .Release.Name }}-scheduler
+rules:
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["delete", "get", "patch", "update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create", "patch", "update"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["delete", "get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["bindings", "pods/binding"]
+    verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["pods/status"]
+    verbs: ["patch", "update"]
+  - apiGroups: [""]
+    resources: ["replicationcontrollers", "services"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["app", "extensions"]
+    resources: ["replicasets"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["apps"]
+    resources: ["statefulsets"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["policy"]
+    resources: ["poddisruptionbudgets"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims", "persistentvolumes"]
+    verbs: ["get", "list", "watch"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Chart.Name }}-{{ .Release.Name }}-scheduler
+subjects:
+- kind: ServiceAccount
+  name: {{ .Chart.Name }}-{{ .Release.Name }}-scheduler
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ .Chart.Name }}-{{ .Release.Name }}-scheduler
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
We can do `name: system:kube-scheduler` instead.

Picked that up from [this presentation](https://www.youtube.com/watch?v=4TaHQgG9wEg&t=14m22s&index=210&list=PLj6h78yzYM2N8GdbjmhVU65KYm_68qBmo).

Reply:
But... if we want to support having replicas >1 of our scheduler we should add stuff to the ClusterRole under resourceNames as described in [configure multiple schedulers - part 3](https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/#3-run-the-second-scheduler-in-the-cluster).